src/content/docs/release-notes/docs-release-notes/docs-6-14-2024.mdx

```diff
@@ -0,0 +1,85 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-07'
+version: 'version: June 7-13, 2024'
+---
+
+### New docs
+
+* [NR24-01 security bulletin](/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01) addresses the recent Fluent Bit security vulnerability.
+* A new [Cacti integration](/docs/infrastructure/host-integrations/host-integrations-list/cacti-integration) gives you information on how to see your RRD data in New Relic.
+
+### Major changes
+
+* We've moved away from the `applied intelligence` name, because it's confusing when compared to our other AI efforts. We completed a massive sweep to remove nearly all mention of "applied intelligence" from our docs.
```

Comment:
```suggestion
* We've moved away from using `applied intelligence` to refer to our Alerts capability. This creates a clearer, more cohesive experience as we introduce more AI features and functionalities. We completed a massive sweep to remove nearly all mention of "applied intelligence" from our docs.
```

Reply:
this is like not great writing but i wanted to flip the rationale to something more positive and user centric

Reply:
i mean that my changes are not great writing* lol please spruce up as needed!!